services/src/handlers/plots.rs

```diff
@@ -135,10 +142,26 @@ async fn get_plot_handler<C: Context>(
         .await
         .context(error::Operator)?;
 
+    // handle request and workflow crs matching
```

Comment:
is this identical to wfs and wms ? if yes, could we make a util method?

Reply:
it is not identical. In WMS/WFS we append a reprojection operator (which reprojects the bbox). Here, we reproject the bbox ourselves (because there is no reprojection operator for plots).